plugins/database/plugin.go

```diff
@@ -103,6 +103,10 @@ func configure(_ *node.Plugin) {
 	runDatabaseGC()
 }
 
+func run(*node.Plugin) {
+	// placeholder
+}
+
```

Comment:
No need to keep unused code

Reply:
This is from #1439
We must have this function so the database will configure itself on time

but maybe I will revert this merge and merge back with develop so it won't appear here

Reply:
I thought we decided to close https://github.com/iotaledger/goshimmer/pull/1439?

Reply:
I think you got confused with #1440

#1439 is already merged and I think it is needed becuase we didn't configure on time..
Maybe there were more changes than what I did that are needed, but my goal was to do the minimum amount of changes to not break anything else

Reply:
Ah I see, I am okay with that then

Reply:
I merged develop to my branch to ensure and I saw all was merged well so I am not bothering with the revert